### PR TITLE
Document agent token permissions

### PR DESCRIPTION
**Pull request by sbquinlan**
## Summary & Motivation

Documenting the agent token permissions dialog for Dagster Cloud.

## How I Tested These Changes

buildkite

### CODE REVIEW

**Comment by sbquinlan** [comment]
Current dependencies on/for this PR:
* `master`
  * **PR #19785** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/19785?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/dagster-io/dagster/19785?utm_source=stack-comment).

File: docs/content/dagster-cloud/account/managing-user-agent-tokens.mdx [hunk 1]
```diff
@@ -27,9 +27,19 @@ Agent tokens are used to authenticate [Hybrid agents](/dagster-cloud/deployment/
...
+- **To edit a token's description**, click the **pencil icon**.
+
+### Modifying agent tokens
+
+1. Sign in to your Dagster Cloud account.
+2. Click the **user menu (your icon) > Organization Settings**.
+3. Click the **Tokens** tab.
+4. Click **Edit** next to the agent token you'd like to change.
+
+The permissions dialog allows you to edit a token's ability to access certain deployments. By default, agent tokens have permission to access any deployment in the organization including branch deployments. This is called "Org Agent" and is set using the toggle in the top right of the dialog. To edit individual deployment permissions, "Org Agent" has to first be toggled off.
```
Comment (erinkcochran87) [inline docs/content/dagster-cloud/account/managing-user-agent-tokens.mdx:40]:
```suggestion
The permissions dialog allows you to edit a token's ability to access certain deployments. By default, agent tokens have permission to access any deployment in the organization including branch deployments. This is called **Org Agent** and is set using the toggle in the top right of the dialog. To edit individual deployment permissions, **Org Agent** has to first be toggled off.
```

UI components (labels, page names, buttons, etc.) should be bolded

File: docs/content/dagster-cloud/account/managing-user-agent-tokens.mdx [hunk 2]
```diff
@@ -10,7 +10,7 @@ In this guide, we'll walk you through creating and revoking user and agent token
 
 ---
 
-## Managing agent tokens
```
Comment (erinkcochran87) [inline docs/content/dagster-cloud/account/managing-user-agent-tokens.mdx:13]: This should remain `Managing`, as the section now includes instructions for two (or three, see my other comment) different things: creating the token, assigning permissions, and revoking it.

We could add a heading to line 22 (I can't comment directly on it, apparently) for token creation: `### Creating agent tokens`

File: docs/content/dagster-cloud/account/managing-user-agent-tokens.mdx [hunk 3]
```diff
@@ -27,9 +27,19 @@ Agent tokens are used to authenticate [Hybrid agents](/dagster-cloud/deployment/
...
+### Modifying agent tokens
+
+1. Sign in to your Dagster Cloud account.
+2. Click the **user menu (your icon) > Organization Settings**.
+3. Click the **Tokens** tab.
+4. Click **Edit** next to the agent token you'd like to change.
+
+The permissions dialog allows you to edit a token's ability to access certain deployments. By default, agent tokens have permission to access any deployment in the organization including branch deployments. This is called "Org Agent" and is set using the toggle in the top right of the dialog. To edit individual deployment permissions, "Org Agent" has to first be toggled off.
+
+- **To revoke a token**, click **Revoke** in the bottom left of the permissions dialog.
```
Comment (erinkcochran87) [inline docs/content/dagster-cloud/account/managing-user-agent-tokens.mdx:42]: This bit about revoking gets kinda lost down here now. I'm not sure how often this comes up, but we can add a section about this to make it easier to find:

```suggestion
### Revoking agent tokens

To revoke a token:

1. Sign in to your Dagster Cloud account.
2. Click the **user menu (your icon) > Organization Settings**.
3. Click the **Tokens** tab.
4. Click **Edit** next to the agent token you'd like to change.
5. Click **Revoke** in the bottom left of the permissions dialog. When prompted, confirm to proceed with revoking the token.
```

File: docs/content/dagster-cloud/account/managing-user-agent-tokens.mdx [hunk 4]
```diff
@@ -27,9 +27,19 @@ Agent tokens are used to authenticate [Hybrid agents](/dagster-cloud/deployment/
 
 After the token is created:
 
-- **To edit a token's description**, click the **pencil icon**.
 - **To view a token**, click **Reveal token**. Clicking on the token value will copy it to the clipboard.
-- **To revoke a token**, click **Revoke**.
+- **To edit a token's description**, click the **pencil icon**.
+
+### Modifying agent tokens
```
Comment (erinkcochran87) [inline docs/content/dagster-cloud/account/managing-user-agent-tokens.mdx:33]: I think it'd be helpful to re-name this section to something about granting permissions - maybe "Assigning agent token permissions" or something along those lines. That's more specific than 'modifying,' and users can only assign permissions or revoke the token in this dialog at the moment anyhow.

**Review by erinkcochran87 — COMMENTED** [review]
ty for this! Let me know if I can clarify anything in my comments.

**Comment by sbquinlan** [comment]
Thanks @erinkcochran87!

File: docs/content/dagster-cloud/account/managing-user-agent-tokens.mdx [hunk 5]
```diff
@@ -20,16 +20,36 @@ In this guide, we'll walk you through creating and revoking user and agent token
 
 Agent tokens are used to authenticate [Hybrid agents](/dagster-cloud/deployment/agents) with the Dagster Cloud Agents API.
 
+## Creating agent tokens
```
Comment (erinkcochran87) [inline docs/content/dagster-cloud/account/managing-user-agent-tokens.mdx:23]:
```suggestion
### Creating agent tokens
```

**Review by erinkcochran87 — APPROVED** [review]
ty! I noticed one heading was an H2 instead of an H3, so I added a comment to it. Address that and then feel free to merge! 🎉

**Pull request merged**